passworddialog.cpp

```diff
@@ -47,9 +57,18 @@ PasswordDialog::~PasswordDialog()
 {
 }
 
+void PasswordDialog::resizeEvent (QResizeEvent *event) {
```

Comment:
I'm not sure about this kind of "homemade" elision...can't this be done "automagically" byt qt (or style) by setting the right properties to `commandL`?